README.md

```diff
@@ -32,7 +32,7 @@ Currently there are no binary distributions of the engine as of yet, and we are
 If you wish to attempt to build a basic visual novel with the existing C++ API, you must first install the following dependencies:
 
 ### Dependencies
-- .NET SDK 5.0.100
+- .NET SDK 6.0.100
```

Comment:
I think we should change this to the RC 1 version as its the readme - I don't think 6.0.100 explicitly exists yet (even though CMake is like "yeah k I'll still work")